classes.py

```diff
@@ -0,0 +1,416 @@
+'''
```

Comment:
For all tables look at adding in created_at, created_by, updated_at, and updated_by